plugins/inputs/ipmi_sensor/README.md

```diff
@@ -4,23 +4,32 @@ Get bare metal metrics using the command line utility `ipmitool`
 
 see ipmitool(https://sourceforge.net/projects/ipmitool/files/ipmitool/)
 
-The plugin will use the following command to collect remote host sensor stats:
+The plugin supports two modes. When 'servers' is not specified, the plugin will utilize the OpenIPMI kernel device driver to retrieve the sensor stats.
```

Comment:
what does this mean? does it still run an ipmitool command? can you provide it here?